pkg/operator/ceph/pool/radosnamespace/controller.go

```diff
@@ -264,6 +264,11 @@ func (r *ReconcileCephBlockPoolRadosNamespace) updateClusterConfig(cephBlockPool
 			RadosNamespace: cephBlockPoolRadosNamespace.Name,
 		},
 		RadosNamespace: cephBlockPoolRadosNamespace.Name,
+		CephFS: &csi.CsiCephFSSpec{
```

Comment:
is cephfs mounts update needed with rbd radosnamspace?

Reply:
> is cephfs mounts update needed with rbd radosnamspace?

well the same cluster id can be used for cephfs storageclass too, so its better to have it.

Reply:
No but looks little miss-leading

Reply:
> No but looks little miss-leading

I dont think so.
This is completely internal so its fine.

Reply:
We are upgrading this with `ReconcileCephFilesystemSubVolumeGroup` so wanted to understand why it is needed with radosnamespace, is this so that we don't lose track of it in the config map? @Rakshith-R

Reply:
> We are upgrading this with `ReconcileCephFilesystemSubVolumeGroup` so wanted to understand why it is needed with radosnamespace, is this so that we don't lose track of it in the config map? @Rakshith-R

These settings(read affinity  + mount options) must be considered default from Cephcluster CR and added when ever you create a new entry,